docs/modules/edge/edged.md

```diff
@@ -43,6 +43,27 @@ This module helps in monitoring pod status for edged. Every second, using probe'
 
 *Fig 5: PLEG at EdgeD*
 
+## CRI for edged
+
+Container Runtime Interface (CRI) – a plugin interface which enables edged to use a wide variety of container runtimes, without the need to recompile and also support multiple runtimes like docker, contianerd, cri-o etc
```

Comment:
Container Runtime Interface (CRI) is a plugin interface.

contianerd  -> containerd